### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Installed `jest` in the repository
- Added operations.test.js to the repository.
- Added testings for `addNew` function in operations.js.
- Added testings for `deleteTask` function in operations.js.

### CODE REVIEW

File: src/tests/mock.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+const localStorage = new LocalStorage();
+const todos = localStorage.getItem('todos');
+
+class Store {
+  static getItems() {
+    return JSON.stringify(localStorage.getItem('todos'));
+  }
+
+  static storeItem(items) {
+    // localStorage.clear('items');
```
Comment (Dipeshtwis) [inline src/tests/mock.js]: - [ ] I would recommend removing commented code as it would make the code cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @rpire and @Kalunge, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Tests are grouped using the describe() method.
:+1: All functions responsible for adding and deleting list items are tested.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] README file should have steps about how to test the project. (Hint: `npm run test`)
- [ ] Currently, the live link pointing to the README itself. Please fix that.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/mock.js (hunk 1 above)
Comment (Kalunge) [reply src/tests/mock.js]: Hello @Dipeshtwis requested changes have been effected

**Review by hemantso — APPROVED** [review]
Hi @rpire and @Kalunge, 

## Your project is complete! 
There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 


Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**